screens/generate_pokemon/dropdown.lua

```diff
@@ -1,135 +1,159 @@
-local gooey = require "screens.generate_pokemon.gooey.gooey"
```

Comment:
I don't quite understand what was going on here. There were 3 excess files under generate_pokemon with many unused functions, all of them only used by this dropdown. They also didn't seem to provide any value at all. I deleted all of them and just used gooey.gooey.